trade-finance-manager-api/src/v1/api.js

```diff
@@ -269,6 +269,39 @@ const submitDeal = async (dealType, dealId, auditDetails) => {
   }
 };
 
+/**
+ * Updates the deal cancellation object on a TFM AIN or MIN deal
+ * @param {object} params
+ * @param {string} params.dealId - deal cancellation to update
+ * @param {Partial<import('@ukef/dtfs2-common').TfmDealCancellation>} params.dealCancellationUpdate - deal cancellation update to make
+ * @param {import('@ukef/dtfs2-common').AuditDetails} params.auditDetails - user making the request
+ * @returns {Promise<import('mongodb').UpdateResult>} update result object
+ */
+const updateDealCancellation = async ({ dealId, dealCancellationUpdate, auditDetails }) => {
+  try {
+    const isValidDealId = isValidMongoId(dealId);
+
+    if (!isValidDealId) {
+      throw new InvalidDealIdError(dealId);
+    }
+
+    const response = await axios({
+      method: 'put',
+      url: `${DTFS_CENTRAL_API_URL}/v1/tfm/deals/${dealId}/cancellation`,
+      headers: headers.central,
+      data: {
+        dealCancellationUpdate,
+        auditDetails,
+      },
+    });
+
+    return response.data;
+  } catch (error) {
+    console.error(error);
+    throw error;
```

Comment:
Thanks @BethThomas141 , but this seems to be a new approach compared to the other functions, there are so many different approaches here 😞 => 

```js
// new approach in this PR
} catch (error) {
  console.error();
  throw error;
}

// existing
} catch ({ response }) {
  console.error();

  return false;
}

// existing
} catch (error) {
  console.error();
  return onError( ... );
}

// existing
} catch (error) {
  console.error();
  return { ... };
}
```

Can we use one of the existing approaches please so that we don't introduce another new approach